submissions/kissik/data-structures/index.js

```diff
@@ -0,0 +1,127 @@
+const http = require('http');
+
+const fs = require('fs');
+
+const port = process.env.PORT || 3210;
+
+const page404 = {
+  path: 'pages/404.html',
+  type: { 'Content-Type': 'text/html' },
+  verb: 404
+};
+
+const action = {
+  type: { 'Content-Type': 'text/html' },
+  verb: 200
+};
+
+const contentType = {
+  '/': {
+    path: 'pages/index.html',
+    type: { 'Content-Type': 'text/html' },
+    verb: 200
+  },
+  '/css/theme.css': {
+    path: 'css/theme.css',
+    type: { 'Content-Type': 'text/css' },
+    verb: 200
+  },
+  '/img/binoculars.jpg': {
+    path: 'img/binoculars.jpg',
+    type: { 'Content-Type': 'image/jpeg' },
+    verb: 200
```

Comment:
```suggestion
    responseCode: 200
```
I believe `responseCode` will better describe value